napari_tiff/_tests/test_tiff_reader.py

```diff
@@ -67,7 +66,6 @@ def test_reader(tmp_path, data_fixture, original_data):
 
 @pytest.mark.parametrize("reader, data_fixture, original_data", [
     (imagecodecs_reader, example_data_filepath, np.random.random((20, 20))),
-    (imagej_reader, example_data_tiff,  np.random.randint(0, 255, size=(20, 20)).astype(np.uint8)),
```

Comment:
It's important that we don't lose functionality, so we'll need to keep this test.

Reply:
...ah, I think I see what's happening with this restructure now

Reply:
See above